### DISCUSSION TITLE
pb avec sambaNTPassword

### DISCUSSION THREAD

**Discussion by caroleb24** [ltb-project/self-service-password#731] [Q&A]
Bonjour,

Le mot de passe de l'annuaire est bien synchronisé mais le sambaNTPassword reste l'ancien. 
Message d'erreur : le mot de passe a été refusé.
Je pense que c'est une option qui manque dans le config.inc.local.php. Voilà ce que j'ai ajouté 👍 
#samba
$samba_mode = true;
$shadow_options['update_shadowLastChange'] = true;
$hash = "clear";
ou peut-être faut-il utiliser un script complémentaire avec la variable $posthook.

pour information, samba, openldap et self-service-password sont installés sur des machines différentes. 

Merci pour votre aide

**Comment by coudot** [comment]
Hello,

check ACL (olcAccess parameter in OpenLDAP) to be sure that SSP service account has the right to write samba attributes
